fix-hosts-file-on-boot/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Copy over the fix-hosts script
+  template:
+    src: "fix-hosts.sh.j2"
+    dest: "{{ chia_root }}/fix-hosts.sh"
+    owner: ubuntu
```

Comment:
Could probably be owned by root and at more of a "system" location, since its not really specific to chia?